exes/mqtt-broadcaster/src/impl.cpp

```diff
@@ -0,0 +1,118 @@
+#include "impl.hpp"
+
+namespace tfc::mqtt::impl {
+
+// Create a timestamp in milliseconds from 1 Jan 1970 according to spec
+auto timestamp_milliseconds() -> std::chrono::milliseconds {
+  using std::chrono::duration_cast;
+  using std::chrono::milliseconds;
+  using std::chrono::system_clock;
+  return duration_cast<milliseconds>(system_clock::now().time_since_epoch());
+}
+
+/// This function converts tfc types to Spark Plug B types
+/// More information can be found (page 76) in the spec under section 6.4.16 data types:
+/// https://sparkplug.eclipse.org/specification/version/3.0/documents/sparkplug-specification-3.0.0.pdf
```

Comment:
```suggestion
```
no need to copy paste comment from header

Reply:
removed